src/plugins/kibana_utils/public/render_complete/index.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import './directive';
+import 'ui/render_complete/directive';
```

Comment:
Lets see if we can remove this line. The fact that it does anything is kind of terrible - this code gets executed when items from this file are imported (I thiiiiink thats right) - at any rate, there's nothing calling this functionality in an expected manner. 

Perhaps we can `import 'ui/render_complete/directive';` in the following places - 
```
src/legacy/core_plugins/kibana/public/discover/directives/index.js
src/legacy/ui/public/vis/vis.js
```

Reply:
I focused too much on passing the test and forgot directives don't have place in np. 

I thought about moving them to the controllers of the angular components. But your idea is better. 

I've applied the change in the following commit.